Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2231,6 +2231,8 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], params: D
                 elif params.get('comment_tag') in tags:
                     key = 'comments'
                 user = entry.get('user', 'dbot')
+                # Sometimes user is an empty str, not None, therefore nothing is displayed in ServiceNow
+                user = 'dbot' if user == '' else user
```

Comment:
This should do the trick
```suggestion
                # Sometimes user is an empty str, not None, therefore nothing is displayed in ServiceNow
                user = entry.get('user', 'dbot') or 'dbot'
```